node/pkg/txverifier/evm.go

```diff
@@ -86,10 +86,18 @@ func (tv *TransferVerifier[ethClient, Connector]) ProcessEvent(
 
 	// Parse raw transaction receipt into high-level struct containing transfer details.
 	transferReceipt, parseErr := tv.ParseReceipt(receipt)
-	if parseErr != nil || transferReceipt == nil {
-		tv.logger.Warn("error when parsing receipt. skipping validation",
-			zap.String("receipt hash", receipt.TxHash.String()),
-			zap.Error(parseErr))
+	if transferReceipt == nil {
```

Comment:
This change assumes that `tv.ParseReceipt` will never return an error when `transferReceipt` is not nil. Is that a safe assumption (maybe now, but what about in the future)?

Reply:
That's a good point, I'll add error-handling at this level at well.

Reply:
Added in the most recent commit.